packages/doenetml-worker-rust/lib-doenetml-core/src/core/graph_node.rs

```diff
@@ -197,10 +221,72 @@ mod iterators {
             }
         }
     }
+
+    /// Iterate through the "content" children of a node. That is,
+    /// all the non-virtual children. If a virtual child is detected,
+    /// it's children are recursively iterated over.
+    ///
+    /// This also returns a boolean indicating whether the node is marked by `mark`.
```

Comment:
`mark` -> `marker`

It returns a boolean indicating whether the iterator encountered virtual node ancestor of the node that is marked by `marker`. (Or something indicating that the node is marked based on its virtual node ancestor.)